src/server/pps/server/worker_rc.go

```diff
@@ -153,11 +177,11 @@ func (kd *kubeDriver) workerPodSpec(options *workerOptions, pipelineInfo *pps.Pi
 		Name:  client.PPSPipelineNameEnv,
 		Value: pipelineInfo.Pipeline.Name,
 	}, {
-		Name:  "LOKI_SERVICE_HOST_VAR",
-		Value: kd.config.LokiHostVar,
+		Name:  "LOKI_SERVICE_HOST",
```

Comment:
Both sets of variables could be included, but I figured that it would be easier for debugging to just include whatever was actually being used.